cdap-common/bin/common.sh

```diff
@@ -232,6 +232,11 @@ set_hive_classpath() {
 
       if [[ $(which hive 2>/dev/null) ]]; then
         HIVE_VAR_OUT=$(hive -e 'set -v' 2>/dev/null)
+        __ret=$?
+        if [ ${__ret} -ne 0 ]; then
+          echo "ERROR - Problem running: hive -e 'set -v'"
```

Comment:
I suggest giving a little more context: ``"Error - explore.enabled but unable to determine Hive classpath - Problem running: hive -e 'set -v'"`` or similar.